scripts/get-changed-tests.mjs

```diff
@@ -112,7 +112,7 @@ export default async function getChangedTests() {
       .catch(() => false)
 
     if (fileExists && file.match(/^test\/.*?\.test\.(js|ts|tsx)$/)) {
-      if (file.startsWith('test/e2e/')) {
+      if (file.startsWith('test/e2e/') || file.startsWith('test/integration')) {
```

Comment:
We only need to run this for one mode not both since they don't differ for dev/prod

Reply:
Not for Turbopack though. Tests sometimes skip if they're run with `test-dev-turbo` or `test-start-turbo` e.g. https://github.com/vercel/next.js/blob/83eb616e95a8de15504f5f49386fefe94d869185/test/integration/scroll-back-restoration/test/index.test.js#L77